src/main/java/com/jhuguet/sb_taskv1/app/controller/GuestController.java

```diff
@@ -33,14 +33,15 @@ public GuestController(JwtUtils jwtUtils, AuthenticationManager manager, UserSer
     }
 
     @PostMapping("login")
-    public String login(@RequestBody AuthenticationRequest auth) throws IOException {
+    public Map<String, String> login(@RequestBody AuthenticationRequest auth) throws IOException, WrongCredentials {
```

Comment:
Why don't you return Dto object, but Map?

Reply:
Changed return type